package.json

```diff
@@ -48,7 +48,7 @@
     "@commitlint/config-conventional": "^12.0.1",
     "@contentful/contentful-extension-scripts": "^0.20.4",
     "@contentful/eslint-config-extension": "0.4.2",
-    "@contentful/f36-components": "^4.9.0",
```

Comment:
r u sure 🙄?

Reply:
This is a revert, I first updated f36 generally and then realized that it may be safer to update it only where I need the newest version. Would it be better instead to update it everywhere?

Reply:
I agree it's safer 👍